test/Microsoft.AspNetCore.Mvc.FunctionalTests/BasicTests.cs

```diff
@@ -617,6 +616,69 @@ public static IEnumerable<object[]> HtmlHelperLinkGenerationData
             Assert.Equal("OnGetTestName", content);
         }
 
+        [Fact]
+        public async Task BindingWorksForPolymorphicTypes()
```

Comment:
Separately, nothing about polymorphic model binding fits in the **Basic** web site.